src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3465,9 +3472,11 @@ export class ResultsViewPageStore {
             ],
             invoke: () => {
                 return Promise.resolve(
-                    _.difference(
-                        this.molecularProfilesInStudies.result!,
-                        this.selectedMolecularProfiles.result!
+                    excludeMutationProfilesForDownload(
```

Comment:
@dippindots The name of this `readonly` variable is `nonSelectedMolecularProfiles`, so it seems strange to me to filter it with `excludeForDownload` here. Maybe more comprehensible to have a separate `readonly` variable called `downloadableMolecularProfiles` that depends on this one. Or something like that